airbyte-integrations/bases/base-normalization/entrypoint.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+TRANSFORMED_CONFIG=transformed_config.json
+DBT_MODEL=dbt_model.json
+set -e
```

Comment:
NIT: put this after the shebang, you never know if someone will change how variable are define and you want to make sure the script fails if it fails